### DISCUSSION TITLE
storybook/next with custom image loader

### DISCUSSION THREAD

**Discussion by atijera-valtech** [storybookjs/storybook#24639] [Help]
Hello! In my team we are building and app using NEXT.js and Storybook, we have a custom image loader in our `next.config.js`, but it is not working within storybook, I would like to know if that is support yet by storybook or if there's any solution or workaround for it, Thanks!

**Comment by woodsboe** [comment]
It is supported.
I've recently worked on a project where we implemented a custom image loader, and it works fine.

Are you using the `@storybook/nextjs` framework in your `.storybook/main.js` config?

EDIT: We don't use the next.config.js, but the loader prop on the `next/image`.
I haven't tried it using the next config file.

**Comment by woodsboe** [comment]
There may also be some info here which could be helpful https://www.npmjs.com/package/@storybook/nextjs#options

**Comment by joevaugh4n** [comment]
Thank you for jumping in here, @woodsboe!
